src/bpmncwpverify/bpmn.py

```diff
@@ -74,6 +91,8 @@ def accept(self, visitor: "BpmnVisitor") -> None:
 class IntermediateEvent(Event):
     def __init__(self, element: Element):
         super().__init__(element)
+        tag = element.tag.partition("}")[2]
+        self.type = "catch" if "Catch" in tag else "throw" if "Throw" in tag else "send"
```

Comment:
this is to see if the intermediate event is a catch or throw event

Reply:
Yes. That is my understanding.